docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,161 @@
+
+For more information on how to configure the test run, see [Command Line Interface](../using-testcafe/command-line-interface.md).
+
+## Viewing the Test Results
+
+While the test is running, TestCafe is gathering information about the test run and outputing the report right into a command shell.
+
+![Test Report](../../images/report.png)
+
+For more information, see [Reporters](../using-testcafe/common-concepts/reporters.md).
+
+## Writing Test Code
+
+### Performing Actions on the Page
+
+Every test should be capable of interacting with a page content. To perform user actions, TestCafe provides a number of [actions](../test-api/actions.md): `click`, `hover`, `typeText`, `setFilesToUpload`, etc.
+They can be called in a chain.
+
+The following fixture contains a simple test that types a developer name into a text editor and then clicks the Submit button.
+
+```js
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('Test1', async t => {
+    await t
+        .typeText('#developer-name', 'John Smith')
+        .click('#submit-button');
+});
+```
+
+All test actions are implemented as async functions of the [test controller object](../test-api/test-code-structure.md#test-controller) `t`. This object is used to access test run API.
+To wait for actions to be complete, use the `await` keyword when calling these actions or action chains.
+
+### Observing Page State
+
+TestCafe allows you to observe the page state.
+For this purpose, it offers special functions: [Selector](../test-api/executing-client-code/index.md#selector-functions) to get direct access to DOM elements and [Client](test-api/executing-client-code/index.md#client-functions) to obtain arbitrary data from the client side.
+You just write your code within these functions, and TestCafe executes the code on the client, thus you will obtain the actual state of the page.
+
+For example, clicking the Submit button on the sample web page opens a "Thank you" page.
+To get access to DOM elements on the opened page, the `Selector` function can be used.
+The following example demonstrates how to access the article header element and obtain its actual text.
+
+```js
+import { Selector } from 'testcafe';
+
+// Declare the parameterized Selector function
+// to get access to a DOM element identified by the `id` attribute
+const getElementById = Selector((id) => document.querySelector(`#${id}`));
+
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('Test1', async t => {
+    await t
+        .typeText('#developer-name', 'John Smith')
+        .click('#submit-button');
+
+    // Use the Selector function to get access to the article header
+    const articleHeader = await getElementById('article-header');
+
+    // Obtain the text of the article header
+    const headerText = articleHeader.textContent;
+});
+```
+
+For more information, see [Executing Client Code](../test-api/executing-client-code.md).
```

Comment:
`Executing Client Code` is about selectors as well as client functions.
I don't like `Executing Client Code` much though...